versioned/tests/src/main/java/org/projectnessie/versioned/tests/AbstractReferences.java

```diff
@@ -160,29 +159,37 @@ public void createAndDeleteTag() throws Exception {
   }
 
   @Test
-  void toRef() throws VersionStoreException {
+  void getNamedRef() throws VersionStoreException {
     final BranchName branch = BranchName.of("toRef");
     store().create(branch, Optional.empty());
     store().hashOnReference(branch, Optional.empty());
 
     final Hash firstCommit = commit("First Commit").toBranch(branch);
 
-    assertThat(store().toRef(branch.getName())).isEqualTo(WithHash.of(firstCommit, branch));
+    assertThat(store().getNamedRef(branch.getName(), GetNamedRefsParams.DEFAULT))
+        .extracting(ReferenceInfo::getHash, ReferenceInfo::getNamedRef)
+        .containsExactly(firstCommit, branch);
 
     final Hash secondCommit = commit("Second Commit").toBranch(branch);
     final Hash thirdCommit = commit("Third Commit").toBranch(branch);
 
     store().create(BranchName.of(thirdCommit.asString()), Optional.of(firstCommit));
     store().create(TagName.of(secondCommit.asString()), Optional.of(firstCommit));
 
-    assertThat(store().toRef(secondCommit.asString()))
-        .isEqualTo(WithHash.of(firstCommit, TagName.of(secondCommit.asString())));
-    assertThat(store().toRef(thirdCommit.asString()))
-        .isEqualTo(WithHash.of(firstCommit, BranchName.of(thirdCommit.asString())));
+    assertThat(store().getNamedRef(secondCommit.asString(), GetNamedRefsParams.DEFAULT))
```

Comment:
Would it be too much trouble to refactor and use dedicated names/variables for branches/tags to avoid confusion with hashes... unless we're trying to assert that hashes can be used as branch names (which does not seems to be the case here)? The code in its current form was a bit hard to understand for me :)

Reply:
Done